pom.xml

```diff
@@ -659,5 +666,74 @@
         </plugins>
       </build>
     </profile>
+
+    <profile>
+      <id>java17-target</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <configuration>
+              <release>17</release>
+              <compilerArgs combine.children="append">
+                <compilerArg>${extraJavaVectorTestArgs}</compilerArg>
+              </compilerArgs>
+            </configuration>
+          </plugin>
+
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+              <argLine>${surefire.argLine} ${extraJavaTestArgs} ${extraJavaVectorTestArgs}</argLine>
+              <systemPropertyVariables>
```

Comment:
Why are these needed?

Reply:
extraJavaVectorTestArgs is only for Java17 Vector module, I have rename it to extraJavaVectorArgs, and move it to profile vector